packages/target-ethers-v5/test/generation.test.ts

```diff
@@ -29,3 +30,122 @@ describe('Ethers generation edge cases', () => {
+        inputs: [
+          {
+            name: undefined,
+            isIndexed: false,
+            type: {
+              type: 'array',
+              itemType: { type: 'address', originalType: 'address' },
+              originalType: 'address[]',
+            },
+          },
+          {
+            name: undefined,
+            isIndexed: false,
+            type: {
+              type: 'array',
+              itemType: { type: 'uinteger', bits: 256, originalType: 'uint256' },
+              originalType: 'uint256[]',
+            },
+          },
+        ],
+      },
+    ]
+
+    const actual = generateEventFilters(events)
+
+    expect(actual).toEqual(expect.stringMatching('arg0?: null'))
+    expect(actual).toEqual(expect.stringMatching('arg1?: null'))
+  })
+
+  it('generates event filters', () => {
+    const abi = ([
+      {
+        anonymous: false,
+        inputs: [
+          { indexed: true, internalType: 'address', name: 'owner', type: 'address' },
+          { indexed: true, internalType: 'address', name: 'approved', type: 'address' },
+          { indexed: true, internalType: 'uint256', name: 'tokenId', type: 'uint256' },
+        ],
+        name: 'Approval',
+        type: 'event',
+      },
+      {
+        anonymous: false,
+        inputs: [
+          { indexed: true, internalType: 'address', name: 'owner', type: 'address' },
+          { indexed: true, internalType: 'address', name: 'operator', type: 'address' },
+          { indexed: false, internalType: 'bool', name: 'approved', type: 'bool' },
+        ],
+        name: 'ApprovalForAll',
+        type: 'event',
+      },
+      {
+        anonymous: false,
+        inputs: [
+          { indexed: true, internalType: 'address', name: 'from', type: 'address' },
+          { indexed: true, internalType: 'address', name: 'to', type: 'address' },
+          { indexed: true, internalType: 'uint256', name: 'tokenId', type: 'uint256' },
+        ],
+        name: 'Transfer',
+        type: 'event',
+      },
+      {
+        anonymous: false,
+        inputs: [
+          { indexed: true, internalType: 'address', name: 'owner', type: 'address' },
+          { indexed: false, internalType: 'uint256', name: 'level', type: 'uint256' },
+          { indexed: false, internalType: 'uint256', name: 'summoner', type: 'uint256' },
+        ],
+        name: 'leveled',
+        type: 'event',
+      },
+      {
+        anonymous: false,
+        inputs: [
+          { indexed: true, internalType: 'address', name: 'owner', type: 'address' },
+          { indexed: false, internalType: 'uint256', name: 'class', type: 'uint256' },
+          { indexed: false, internalType: 'uint256', name: 'summoner', type: 'uint256' },
+        ],
+        name: 'summoned',
+        type: 'event',
+      },
+    ] as any) as RawAbiDefinition[]
+
+    const contract = parse(abi, 'Rarity')
+    const actual = Object.values(contract.events).map(generateEventFilters)
+
+    const expected = [
```

Comment:
i am not sure if we need such tests as we treat generated typings as a sort of snapshot tests.

Reply:
I added this one because it's easier to add `it.only` inside of Mocha test to work on a localized problem and the test descriptions document intention.

I can remove it if you'd prefer.

Reply:
I think it will easily break once we change anything. I would rather delete it as it doesn't provide any additional value to the snapshot test.